src/web/components/ArticleContainer.tsx

```diff
@@ -25,11 +25,17 @@ const articleAdStyles = css`
         min-height: 274px;
         text-align: center;
         position: relative;
+        @media print{
+            display: none !important;
+        }
     }
     .ad-slot--mostpop {
         ${from.desktop} {
             margin: 0;
             width: auto;
+            @media print{
+            display: none !important;
+        }
```

Comment:
looks like `prettier` is not working here. Try have a look at your IDE to try fix it and run on save. If you are using VSC look at redownloading it (that fixed it for me)

Reply:
are we happy removing ADs on print? @jfsoul @gtrufitt ?

Reply:
Yup